src/App.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import Typography from './components/atoms/Typography/Typography';
```

Comment:
make index file to atoms

Reply:
I created this file.
Just forgot to use it. Will fix in next commit.